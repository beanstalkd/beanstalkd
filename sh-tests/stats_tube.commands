use tubea
put 0 0 0 0

delete 1
stats-tube tubea
stats-tube default
quit
