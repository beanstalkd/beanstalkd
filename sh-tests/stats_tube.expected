USING tubea
INSERTED 1
DELETED
OK 263
---
name: tubea
current-jobs-urgent: 0
current-jobs-ready: 0
current-jobs-reserved: 0
current-jobs-delayed: 0
current-jobs-buried: 0
total-jobs: 1
current-using: 1
current-watching: 0
current-waiting: 0
cmd-delete: 1
cmd-pause-tube: 0
pause: 0
pause-time-left: 0

OK 265
---
name: default
current-jobs-urgent: 0
current-jobs-ready: 0
current-jobs-reserved: 0
current-jobs-delayed: 0
current-jobs-buried: 0
total-jobs: 0
current-using: 0
current-watching: 1
current-waiting: 0
cmd-delete: 0
cmd-pause-tube: 0
pause: 0
pause-time-left: 0

